zebracrossing/campaigns/models.py

```diff
@@ -33,6 +33,21 @@ def __str__(self):
         )
 
 
+class BookedDay(models.Model):
```

Comment:
This PR shouldn't be creating a BookedDay. Have you rebased this branch off main?

Reply:
Yes I did rebase off main.

Reply:
Then remove this second BookedDay class